essential-tests/src/testFixtures/kotlin/org/stellar/anchor/platform/e2etest/Sep6End2EndTest.kt

```diff
@@ -120,6 +122,14 @@ open class Sep6End2EndTest : AbstractIntegrationTests(TestConfig()) {
         COMPLETED,
       )
     assertWalletReceivedStatuses(deposit.id, expectedStatuses)
+
+    val expectedCustomerStatuses =
+      listOf(
+        Sep12Status.ACCEPTED, // initial customer status before SEP-6 transaction
+        Sep12Status.NEEDS_INFO, // SEP-6 transaction requires additional info
+        Sep12Status.ACCEPTED // additional info provided
+      )
+    assertWalletReceivedCustomerStatuses(customer.id, expectedCustomerStatuses)
```

Comment:
Should it not be a state automata? (we always know the sequence of statuses)

Reply:
Good point. It should use an `assertEquals` instead.